src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(/images/star.svg);
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+}
+
+.stars--4 > .stars__star:nth-child(-n+4) {
```

Comment:
all modifiers must work (stars--1, 2 ...). And try to use [cycle](https://sass-lang.com/documentation/at-rules/control/for)

Reply:
edited